cmd/wallet-sdk-gomobile/api/api.go

```diff
@@ -23,22 +27,22 @@ type JSONArray struct {
 
 // KeyHandle represents a public key with associated metadata.
 type KeyHandle struct {
-	PubKey []byte `json:"key,omitempty"` // Raw bytes
-	KeyID  string `json:"keyID,omitempty"`
+	KeyID string   `json:"keyID,omitempty"`
```

Comment:
I see you get the KeyID from `JWK` now in the ID() method - do we still need the `KeyID` field? If not, could we simplify this right down to a `JSONWebKey`?

Reply:
Good point

Reply:
Updated!